spec/requests/message_handler_spec.rb

```diff
@@ -1,7 +1,30 @@
 require 'rails_helper'
 
 describe "Logging API requests" do
-  context "Page Creation" do
+  let(:page) { Page.create(title: 'Foo', slug: 'foo-bar') }
+
+  context "Updating Page Resource" do
+    let(:params) do
+      { type: 'update',
+        uri: "https://act.sumofus.org/rest/v1/petitionpage/8769/",
+        params: {
+          title: 'Foo Bar',
+          tags: ["/rest/v1/tag/878/"]
+        }
+      }
+    end
+
+    context "204 response" do
+      it "updates resource" do
+        VCR.use_cassette('update_resource_204') do
```

Comment:
what are you going for with `use_cassette`? I know that it's not necessary - you can just do 
```
it "updates resource" ,:vcr do
 ...
end
```
and it will create a cassette just for that test. but is there a different functionality you're going for? I don't see you reusing these casettes...